src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_help.py

```diff
@@ -20,6 +23,89 @@
     short-summary: (PREVIEW) Manage Azure IoT hubs.
 """
 
+helps['iot hub certificate'] = """
+    type: group
+    short-summary: Manage IoT Hub certificates.
+"""
+
+helps['iot hub certificate create'] = """
+    type: command
+    short-summary: Create/upload an Azure IoT Hub certificate.
```

Comment:
Why is this command `iot hub certificate create` when `--path` to file containing the certificate is required?
Looks like this should be `iot hub certificate upload`?

Reply:
It is a bit strange. The command creates a Certificate object in a users IoT Hub. One of the parameters for creation is a Unicode string that represents the certificate. It's not truly uploading a file, but rather we are extracting the file contents if it's valid Unicode and using that as the "certificate" parameter, or base64 encoding the contents and using that string as the param. Additionally, we were trying to keep command name parity with the other operations we own.

Reply:
Okay I understand.